nilearn/conftest.py

```diff
@@ -65,49 +62,6 @@ def pytest_configure(config):  # noqa: ARG001
         matplotlib.use("Agg", force=True)
 
 
-@pytest.fixture(autouse=True)
-def no_int64_nifti(monkeypatch):
```

Comment:
We are going to need to remove this fixture if we want to tests things properly in free threading because otherwise it means that ALL our tests are running with this fixture  that relies on monkeypatch that makes it impossible to test things in parallel